data/autoyast_sle12sp2/bug-876411_sles12_btrfs_h5_autoinst.xml

```diff
@@ -882,7 +882,6 @@ find / -name YaST2-Second-Stage.service
       <package>libdnet1</package>
       <package>libgirepository-1_0-1</package>
       <package>libproxy1-config-gnome3</package>
-      <package>libpyglib-gi-2_0-python2-0</package>
```

Comment:
if you adapt it to sp3, there is no point in having them in the sle12sp2 directory and add the sp2 update channel. I really wonder why we have them this way :(